packages/app/src/server/routes/login.js

```diff
@@ -120,14 +122,9 @@ module.exports = function(crowi, app) {
           return res.redirect('/register');
         }
 
-        User.createUserByEmailAndPassword(name, username, email, password, undefined, async(err, userData) => {
+        UserRegistrationOrder.createUserRegistrationOrder(name, username, email, password, undefined, async(err, userData) => {
```

Comment:
Do not edit this method.
This is very old code that can be degraded and is risky.